content/webapp/components/GuideStopCard/index.tsx

```diff
@@ -48,13 +50,26 @@ const GuideStopCard: FunctionComponent<Props> = ({
   type,
   image,
 }) => {
+  const { viewTransitions } = useToggles();
+
+  const CardOuterComponent:
+    | typeof CardOuterTransition
+    | (typeof CardOuter & { href: string }) = viewTransitions
+    ? CardOuterTransition
+    : CardOuter;
```

Comment:
🤙

Reply:
I had to cop out and went for the generic ReactElement solution to get it working and retain my sanity